pkg/asset/kubeconfig/kubeconfig.go

```diff
@@ -104,11 +83,18 @@ func (k *Kubeconfig) Generate(parents map[asset.Asset]*asset.State) (*asset.Stat
 		return nil, errors.Wrap(err, "failed to Marshal kubeconfig")
 	}
 
+	var kubeconfigName string
```

Comment:
just change this to:
```go
var kubeconfigName string
switch k.userName {
case KubeconfigUserNameAdmin:
    kubeconfigName = "kubeconfig"
default:
		kubeconfigName = fmt.Sprintf("kubeconfig-%s", k.userName)
}
```
All kubeconfig files except for admin can be named `kubeconfig-<username>`

Reply:
Done.